translations/translationtool/src/translationtool.php

```diff
@@ -302,12 +313,45 @@ private function createFakeFileForAppInfo() {
 			$content .= '$l->t(' . $this->escape($string) . ');' . PHP_EOL;
 		}
 		
-		file_put_contents($this->dummyFileName, $content);		
+		file_put_contents($this->fakeAppInfoFile, $content);
+	}
+
+	private function createFakeFileForVueFiles() {
+		$fakeFileContent = '';
+
+		foreach ($this->findTranslatableFiles(['.vue']) as $vueFile) {
+			$vueSource = file_get_contents($vueFile);
+			if ($vueFile === false) {
+				echo "Warning: could not read " . $vueFile . PHP_EOL;
+				continue;
+			}
+
+			// t
+			preg_match_all("/t\s*\('([_a-zA-Z]+)',\s*'(.+)'/", $vueSource, $singleQuoteMatches);
```

Comment:
This also matches all methods that end on `t` with two string arguments. Maybe we should check for any non-character element right before that to avoid those false positives as well.

Reply:
Something like `\Wt` should do the trick and works for mail without problems.

Reply:
Makes sense and still matches the same strings for Mail